src/S3/Result/GetObjectAclOutput.php

```diff
@@ -56,7 +56,7 @@ protected function populateResult(ResponseInterface $response, ?HttpClientInterf
             'ID' => $this->xmlValueOrNull($data->Owner->ID, 'string'),
         ]);
         $this->Grants = [];
-        foreach ($data->Grants as $item) {
+        foreach ($data->Grant as $item) {
```

Comment:
I can’t understand why this is changed. Does $data->Grant even exist?

It is probably correct, it is just my ignorance.

Reply:
this is because of https://github.com/async-aws/aws/pull/97/files#diff-bcf76e97ba120d21d9167a6e0778079dR497

AWS returns `<xml><Grant></Grant</xml>` while the property is named `Grants`. This information is defined in the `locationName` property of the shape

Reply:
Thank you